source/style-guide/information-types/prototypes/reference.txt

```diff
@@ -27,6 +27,31 @@ contents, often called a short description. The short description
 explains a little about how the reference material is used, and may link
 to a concept or task page.
 
+Compatibility 
+-------------
+
+Optional. 
+
+A list of MongoDB deployment types that support the referenced method, operator, 
+or database command: 
+
+- `MongoDB Atlas <https://docs.atlas.mongodb.com/>`_: The fully managed service 
+  for MongoDB deployments in the cloud
+
+.. note:: 
+
+   If the command is supported on Atlas, specify whether it's available in all 
```

Comment:
Suggestion:

A few suggestions:
1) I think this has been included on several pages that aren't necessarily focused on a specific command, so maybe it would be helpful to generalize.
2) I think listing the possibilities could limit the interpretation to which it might apply, so perhaps introducing it as an incomplete list could be helpful.
3) Since the possibilities are highly related, I think a bulleted list would be easier to read.

```suggestion
   If Atlas supports the command or functionality, specify a level of compatibility such as one of the following:
   
      - Available in all clusters
      - Available only in certain clusters
      ...
```

Reply:
Gotcha! Thanks for the suggestion.